wazo_acceptance/helpers/schedule.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 The Wazo Authors  (see the AUTHORS file)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class Schedule:
+
+    def __init__(self, context):
+        self._context = context
+        self._confd_client = context.confd_client
+
+    def create(self, name, timezone, tables, destination=None):
```

Comment:
[open discussion] The logic linked to the `tables` object is hardcoded with the create step. So I will moved all logic to the step file and not have a behave object passed to the helper.